src/main/java/frc/robot/subsystems/Conveyor.java

```diff
@@ -1,22 +1,38 @@
 /* ==================================================
- * Authors:
- *
+ * Authors: Fawaaz Kamali Siddiqui, Lemi Miyu, 
+ * Muhammad Ahmad Aniq Nomaan
  * --------------------------------------------------
- * Description:
- *
+ * Description: Contains functions and objects to control
+ * the conveyor susbystem. It also contains code to prevent
+ * jams from ocurring inside the ball storage bay.
  * ================================================== */
 
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Timer;
 
 public class Conveyor extends SubsystemBase
 {
   // HARDWARE //
-
+  private PWMSparkMax conveyorMotor;
+  private DigitalInput intakeSwitch;
+  private DigitalInput conveyorSwitch;
+  private DigitalInput shooterSwitch;
+  private Timer timer;
```

Comment:
@fawaazkamali I'm not sure if there will be switches on the robot this year, did you ask the build team?

Reply:
I did ask somebody from the build team (can't remember his name) about how the conveyors worked and he said that they would be using switches so I just went along with it.

Reply:
Okay. Why did you add the intake and shooter switches in the conveyor?

Reply:
Yeah not sure how many switches they are gonna have this year, but for now I'd say we can leave out the switch code.  If we do eventually add switches then we can come back and modify the code, but if we leave it in now without any switches actually attached, the conveyor may act kinda funky.

Reply:
I would say to test the conveyor, you can initially bind it to run off of one of the buttons on the controller, it can be a separate button for now, to test shooter and conveyor separately, but afterwards, you will probably have conveyor running in standby in the background (triggered by limit switches) and then run the conveyor and shooter at the same time when you press the shoot button to feed the balls in while the shooter is running.

Reply:
I think the intake and the conveyor will be fairly detached this time around since they have a "hopper" kind of thing in between, so they should be controlled independently.